src/Mentat.Repository/Services/CardService.cs

```diff
@@ -109,6 +110,15 @@ public void SaveCard(string id, Card card)
             else
             {
                 card.Id = Guid.NewGuid().ToString();
+                if (card.IsPrivate == true)
+                {
+                    card.IsCustom = true;
+                }
+                if (card.IsPrivate == false)
+                {
+                    card.IsCustom = false;
```

Comment:
See the above comment. I don't understand why you would tie a card's custom status to its privacy status.